yimialmonte/smart-offers-rest-service
# Sors/40 upload images

# What changed:
- New endpoint for upload image file

# How to Test / Acceptance Criteria:
- In postman 
- POST => http://localhost:3000/v1/upload
- Body -  form-data - key = image and choose a image file
-The response should be the image URL

# Ticket(s): 
- 40

# checklist
- [] Tests
- [] Documentation

# Additional Info
- This PR is part of the create offers ticket
